.readthedocs.yaml

```diff
@@ -1,11 +1,18 @@
 # Required by Read The Docs
 version: 2
 
-formats: all
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.8"
```

Comment:
Lets go to Python 3.11 as well, 3.8 is close to EOL, at 3.11 appears to work OK.